src/Tests/Builder/AuthnResponseBuilderTest.php

```diff
@@ -43,7 +41,7 @@ public function testConstructorWithDefaultValue()
         self::assertInstanceOf(\DateTime::class, $authResponse->getIssueInstant());
 
         self::assertEquals($now->getTimestamp(), $authResponse->getIssueInstant()->getTimestamp(), '', 0.5);
-        self::assertEquals($now->getTimezone(), new \DateTimeZone('UTC'));
+//        self::assertEquals(new \DateTimeZone('UTC'), $now->getTimezone());
```

Comment:
My TimeZone in the php.ini is different to UTC. And what is sense to check timezone from the new Datetime and new DatetimeZone ?